### PR TITLE
Augmentation investigation

### PR DESCRIPTION
**Pull request by alicjak1519**

### CODE REVIEW

File: datasets/dataset.py [hunk 1]
```diff
@@ -367,8 +369,18 @@ def _get(
...
+            img_list_to_tensor = ImglistToTensor()
+            images = img_list_to_tensor(images)
+
+            transformed_images = []
+            for image in images:
+                random.seed(transforms_seed)
+                transformed_image = self.transform(image=np.moveaxis(image.numpy(), 0, -1))
+                transformed_images.append(transformed_image)
+
+            images = torch.stack([image['image'] for image in transformed_images])
```
Comment (alicjak1519) [inline datasets/dataset.py:383]: Albumentations seem to not work with 3D images. That is why this funny `random.seed(transforms_seed)` was applied. It enables to repeat exactly the same augmentation for each frame in the subset. Is it ok?

File: datasets/transforms_creator.py [hunk 2]
```diff
@@ -1,8 +1,7 @@
 from typing import Dict
 
-import torchvision.transforms as T
-
-from datasets.dataset import ImglistToTensor
+import albumentations as A
```
Comment (alicjak1519) [inline datasets/transforms_creator.py:3]: According to @AgaMiko and @aga0 suggestion I replaced torchvision transforms with the albumentations lib.

File: train_config.yml [hunk 3]
```diff
@@ -11,20 +11,22 @@ general_parameters:
     neptune: False
 
 augmentations_parameters:
-    apply_resize: True
-    apply_center_crop: True
```
Comment (alicjak1519) [inline train_config.yml:15]: This parameters are removed from the config because they have to be applied for each dataset to make input "readable" for the network.

File: datasets/transforms_creator.py [hunk 4]
```diff
@@ -15,11 +14,10 @@ def __init__(self, augmentations_parameters: Dict = None):
         Args:
             augmentations_parameters (Dict): Dict containing parameters regarding currently used augmentations.
                 [Warning] Must contain fields:
-                    - "apply_resize" (bool)
-                    - "apply_center_crop" (bool)
-                    - "apply_random_erasing" (bool)
```
Comment (alicjak1519) [inline datasets/transforms_creator.py:20]: I removed random_erasing. According to the training results, it doesn't improve metrics. What is more, images with this erasing look doubtfully for human eye.

**Comment by alicjak1519** [comment]
I would like to merge this PR. 
According to the experiments (mentioned in excel with pieces of training results), changing augmentations doesn't improve metrics. On the other hand, these ones (with their default parameters) look more reasonable in the visualization. 
I add a couple of new augmentations with color modification (suggested by @karolmajek) and improved its parametrization in the config file. I hope it will help in the next steps and during the fine-tuning of the model. :)

**Review by adrianlachowicz — APPROVED** [review]

**Review by polquant — APPROVED** [review]

**Review by majsylw — APPROVED** [review]
